File: code-coaches/easy/easter-eggs/EasterEggs.cs
using System;

public class EasterEggs
{
	static void Main(string[] args) {
		int totalEggs = int.Parse(Console.ReadLine());
		int eggsFoundByMe = int.Parse(Console.ReadLine());
		int eggsFoundByFriend = int.Parse(Console.ReadLine());
		int eggsFound = eggsFoundByMe + eggsFoundByFriend;

        if (eggsFound == totalEggs) {
	    	Console.WriteLine("Candy Time");
        } else {
	    	Console.WriteLine("Keep Hunting");
        }
    }
}

File: code-coaches/easy/easter-eggs/EasterEggs.swift
func main() {
    let totalEggs = Int(readLine()!)!
    let eggsFoundByMe = Int(readLine()!)!
	let eggsFoundByFriend = Int(readLine()!)!
		
	let eggsFound = eggsFoundByMe + eggsFoundByFriend
	if eggsFound == totalEggs {
		print("Candy Time")
	} else {
		print("Keep Hunting")
	}
}

main()